receiver/hostmetricsreceiver/internal/scraper/memoryscraper/memory_scraper.go

```diff
@@ -29,18 +31,32 @@ const metricsLen = 1
 
 // scraper for Memory Metrics
 type scraper struct {
-	config *Config
+	config    *Config
+	mb        *metadata.MetricsBuilder
+	startTime pdata.Timestamp
```

Comment:
Looks like `startTime` is redundant field

Reply:
Fixed.